_includes/sqlstyle.guide.md

```diff
@@ -2,21 +2,16 @@
 
 ## Overview
 
-You can use this set of guidelines, [fork them][fork] or make your own - the
-key here is that you pick a style and stick to it. To suggest changes
-or fix bugs please open an [issue][] or [pull request][pull] on GitHub.
+These are guidelines to help you write SQL queries that will be easier to read.
 
-These guidelines are designed to be compatible with Joe Celko's [SQL Programming
-Style][celko] book to make adoption for teams who have already read that book
-easier. This guide is a little more opinionated in some areas and in others a
-little more relaxed. It is certainly more succinct where [Celko's book][celko]
-contains anecdotes and reasoning behind each rule as thoughtful prose.
+Remember that even if you hate a given style at first, generally speaking it is
+far more important that we have _any_ agreed upon style than that we all like it.
 
-It is easy to include this guide in [Markdown format][dl-md] as a part of a
-project's code base or reference it here for anyone on the project to freely
-read—much harder with a physical book.
+**Queries submitted to the Data Engineering or Data Science teams _must_ follow
+the style guide.**  Reading queries is tough enough already without figuring out
```

Comment:
This seems like unnecessary editorializing. We can phrase it along the lines of:

> Queries submitted to the Data teams should follow the style guide, and questions starkly contrasting to, or ignorant of, these guidelines may be asked to be reformatted and resubmitted. Feel free to ask about style rationale, or pose a question how you can make your query (or often taking a step back, question) adhere.